contracts/deprecated/samples/SampleNonAssemblySinglePriceAware.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.2;
+
+import "../message-based/NonAssemblySinglePriceAware.sol";
+import "../../mocks/MockStatePriceProvider.sol";
+
+contract SampleNonAssemblySinglePriceAware is NonAssemblySinglePriceAware {
+  
+  MockStatePriceProvider mockStatePriceProvider = new MockStatePriceProvider();
+
+
+  function execute(bytes32 symbol) public returns(uint256) {
+    return getAssetPrice(symbol);
+  }
+
+
+  function executeWithPrice(bytes32 symbol) public returns(uint256) {
+    return getAssetPrice(symbol);
```

Comment:
I think we should get price from message in here, to compare with other PriceAware versions.